apis/logging/v1/cluster_log_forwarder_types.go

```diff
@@ -45,6 +45,11 @@ type ClusterLogForwarderSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Forwarder Outputs",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:forwarderOutputs"}
 	Outputs []OutputSpec `json:"outputs,omitempty"`
 
+	// Filters are applied to log records passing through a pipeline.
+	// There are different types of filter that can select and modify log records in different ways.
+	// See [FilterTypeSpec] for a list of filter types.
+	Filters []FilterSpec `json:"filters,omitempty"`
```

Comment:
Audit says this enables filters.  Maybe mention they are only valid for pipelines that enable audit and they only apply to audit. To simplify coding do we maybe need to reject pipelines that enable filters and define more inputs then audit?

Reply:
The filter actually can apply to logs from any source, it will ignore non-audit logs. It doesn't make sense to use on a pipeline unless it can receive audit logs but it's not destructive, and I think enforcing the restriction is more complexity.

Reply:
Enforcing is a simple check on our validate code to reject but I defer to you here.  I'm looking to avoid the confusion of "why does my filter not applied to application log message"

Reply:
Gotcha, will review that and either enforce or document.

Reply:
I don't see mention of applying to audit only though I defer to you if it is otherwise clear in the api